v63004/u_inref/gtm8922.csh

```diff
@@ -26,6 +26,8 @@ $MUPIP set -access_method=BG -file a.dat >>& dbcreate_log.txt
 $MUPIP set -access_method=BG -file b.dat >>& dbcreate_log.txt
 
 $MUPIP set -file mumps.dat -flush_time=1:0:0 # Prevent interruptions from flush timers
+$MUPIP set -file a.dat -flush_time=1:0:0 # Prevent interruptions from flush timers
```

Comment:
$MUPIP set -reg "*" should do it on all regions instead of 3 different commands.